site/_posts/2022-07-15-remix-demo.md

```diff
@@ -50,10 +51,10 @@ File-based session storage has one drawback that is only relevant at a certain s
 For this example, you will also need a working FusionAuth instance. There are [various ways](/docs/v1/tech/installation-guide/fusionauth-app) to install FusionAuth depending on your system, but the easiest way is to use Docker and Docker Compose. [Instructions are here](/docs/v1/tech/installation-guide/docker). Assuming you have Docker installed, start FusionAuth by running these commands:
 
 ```bash
-curl -o docker-compose.yml https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml
+curl -o docker compose.yml https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml
```

Comment:
Did you really want to remove the dash here?

Reply:
Please use `{% include posts/install-fusionauth.md %}` (you may need to merge master)

Reply:
Done